app/src/main/java/com/example/taskplanner/data/mapper/project/ProjectDataMapper.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.taskplanner.data.mapper.project
+
+interface ProjectDataMapper<PROJECT_DTO, PROJECT_DOMAIN> {
+    fun dtoListToDomainList(dto: List<PROJECT_DTO>): List<PROJECT_DOMAIN>
```

Comment:
ჯობია რო მაპერი იყოს აბსტრაქტული კლასი და ლისტის გადამეპვის იმპლემენტაცია ხელით დაწერო ანუ ფუნქცია ხო გაქ dtoToDomain რომელიც გიბრუნებს დომეინ მოდელს dto-დან ხოდა იმპლემენტაცია რაღაც ესეთი იქნება 
return dtoList.map { dtoToDomain(it) }

Reply:
done